docs/app/pages/components/components-sections/input-controls/radio-button/radio-button.component.html

```diff
@@ -102,6 +102,15 @@
     the radio button with a <code>name</code> or <code>id</code> if required.
 </p>
 
+<h4>Keyboard Accessibility</h4>
+<p>
+    Due to an issue within native HTML it is impossible to focus any of the
```

Comment:
This sentence is a bit confusing, also HTML is a spec so it's more on the browsers. You can just say that it's not possible to land keyboard focus on a radio button group when the selection option is disabled.